src/jsonpatcherproxy.js

```diff
@@ -143,7 +144,9 @@ var JSONPatcherProxy = (function() {
         return Reflect.deleteProperty(target, key);
       }
     });
-    return proxy;
+    /* keeping track of all the proxies to be able to revoke them later */
+    this.proxifiedObjectsMap.push(proxy);
+    return proxy.proxy;
   };
   //grab tree's leaves one by one, encapsulate them into a proxy and return
   JSONPatcherProxy.prototype._proxifyObjectTreeRecursively = function(
```

Comment:
Hmm... this is a philosophical question now :) I am not sure. Maybe let's see how it goes with controlling it from Palindrom.